### DISCUSSION TITLE
Add a high-contrast monochrome theme

### DISCUSSION THREAD

**Discussion by Soundtoxin** [monkeytypegame/monkeytype#2043] [Ideas]
I didn't see one. Seems like a no-brainer. Pure `#000000` bg and `#ffffff` text. It would be especially nice because of a majority of the current themes being extremely low-contrast. The ones that are commonly used in text editors like solarized dark/light and monokai are among the better options, but I still prefer the pure and sharp colors of simple black and white in most cases.

**Comment by unrealapex** [comment]
I can see that happening. The default for entirely all computers and applications is a white background with dark text. I could create a "default" light theme.

EDIT: Monketype has a dark theme, no light one yet.

**Reply by unrealapex** [reply]
Paper is the closest to what you want, but doesn't incorporate pure white or black.

**Reply by Soundtoxin** [reply]
Dark themes are a lot more popular these days, sometimes even default. Which existing dark theme uses pure black and white? I looked through them before posting this thread and none of them seemed quite right from what I remember.

**Reply by unrealapex** [reply]
Try the `dark` theme.

**Reply by Soundtoxin** [reply]
That's not a pure black background at all. :/
Maybe phone settings names could be followed and a new "AMOLED" or "OLED" theme could be added with proper pure blacks. That, or make a new theme called "black" instead of "dark".

**Comment by unrealapex** [comment]
#2050

**Comment by Miodec** [comment]
For specialty themes like this, you could use the custom theme feature.

**Reply by Soundtoxin** [reply]
Sure, but I linked a friend the site and his first comment was stuff being low contrast. Seems like a simple enough thing to have two high contrast themes (light and dark).

**Reply by unrealapex** [reply]
I agree. I think it would be a bit easier on the user instead of needing to creating one manually. @Miodec is it possible to share custom themes?

**Reply by unrealapex** [reply]
Maybe have a custom theme library like for quotes?

**Reply by Miodec** [reply]
You can click the share button to get a url for the theme
